src/control/cmd/dmg/pretty/pool.go

```diff
@@ -85,3 +86,166 @@ func PrintPoolCreateResponse(pcr *control.PoolCreateResp, out io.Writer, opts ..
 
 	return err
 }
+
+func poolListCreateRow(pool *control.Pool) txtfmt.TableRow {
+	name := pool.Label
+	if name == "" {
+		name = pool.UUID
+	}
+
+	// display size of the largest non-empty tier
+	var sizeTier *control.PoolTierUsage
+	for ti := len(pool.Usage) - 1; ti >= 0; ti-- {
+		if pool.Usage[ti].Size != 0 {
+			sizeTier = pool.Usage[ti]
+			break
+		}
+	}
+	sizeStr := fmt.Sprintf("%s (%s)", humanize.Bytes(sizeTier.Size), sizeTier.TierName)
+
+	// display usage of the most used tier
+	var usedTierName string
+	var usedPercentage int
+	for ti := 0; ti < len(pool.Usage); ti++ {
+		t := pool.Usage[ti]
+		u := float64(t.Size-t.Free) / float64(t.Size)
+
+		if int(u*100) > usedPercentage {
+			usedTierName = t.TierName
+			usedPercentage = int(u * 100)
+		}
+	}
+	usedStr := fmt.Sprintf("%d%% (%s)", usedPercentage, usedTierName)
+
+	// display imbalance of the most imbalanced tier
+	imbalancedTier := pool.Usage[0]
+	for ti := 0; ti < len(pool.Usage); ti++ {
+		if pool.Usage[ti].Imbalance > imbalancedTier.Imbalance {
+			imbalancedTier = pool.Usage[ti]
+		}
+	}
+	imbalanceStr := fmt.Sprintf("%d%% (%s)", imbalancedTier.Imbalance, imbalancedTier.TierName)
+
+	row := txtfmt.TableRow{
+		"Pool":      name,
+		"Size":      sizeStr,
+		"Used":      usedStr,
+		"Imbalance": imbalanceStr,
+		"Disabled":  fmt.Sprintf("%d/%d", pool.TargetsDisabled, pool.TargetsTotal),
+	}
+
+	return row
+}
+
+func printListPoolsResp(out io.Writer, resp *control.ListPoolsResp) error {
+	if len(resp.Pools) == 0 {
+		fmt.Fprintln(out, "no pools in system")
+		return nil
+	}
+
+	formatter := txtfmt.NewTableFormatter("Pool", "Size", "Used", "Imbalance", "Disabled")
+
+	var table []txtfmt.TableRow
+	for _, pool := range resp.Pools {
+		table = append(table, poolListCreateRow(pool))
+	}
+
+	fmt.Fprintln(out, formatter.Format(table))
+
+	return nil
+}
+
+func addVerboseTierUsage(row txtfmt.TableRow, usage *control.PoolTierUsage) txtfmt.TableRow {
+	row[usage.TierName+" Size"] = humanize.Bytes(usage.Size)
+	row[usage.TierName+" Used"] = humanize.Bytes(usage.Size - usage.Free)
+	row[usage.TierName+" Imbalance"] = fmt.Sprintf("%d%%", usage.Imbalance)
+
+	return row
+}
+
+func poolListCreateRowVerbose(pool *control.Pool) txtfmt.TableRow {
+	label := pool.Label
+	if label == "" {
+		label = "-"
+	}
+
+	svcReps := "N/A"
+	if len(pool.ServiceReplicas) != 0 {
+		rl := system.RanksToUint32(pool.ServiceReplicas)
+		svcReps = formatRanks(rl)
+	}
+
+	row := txtfmt.TableRow{
+		"Label":    label,
+		"UUID":     pool.UUID,
+		"SvcReps":  svcReps,
+		"Disabled": fmt.Sprintf("%d/%d", pool.TargetsDisabled, pool.TargetsTotal),
+	}
+
+	for _, tu := range pool.Usage {
+		row = addVerboseTierUsage(row, tu)
+	}
```

Comment:
This is probably going to get unwieldy pretty quickly with > 2 tiers. I wouldn't block on fixing this, but one thought that occurs to me as a way to future-proof this would be to append each tier to a new indented line below the main line, e.g. something like this:
```
Label UUID                                 SvcReps Disabled
----- ----                                 ------- --------
pool  5a9c8adb-44dd-402e-9a67-e5ec9b0d29fd 0       2
    SCM Size SCM Used SCM Imbalance
    -------- -------- -------------
    240 MB   1.9 kB   0%

    NVME Size NVME Used NVME Imbalance
    --------- --------- --------------
    4GB       10 kB     0%
```

I kind of like that better even with 2 tiers, actually. What do you think?

Hmm. I guess this is probably not easy to do with the current txtfmt table implementation. Well, unless something occurs to you, I'm fine with deferring on this to a later release where we add support for > 2 tiers.

Reply:
What about something like this? Probably easier to implement, but I think will still require some modifications to txtfmt or at least using it differently:
```
Label UUID                                 SvcReps Disabled
----- ----                                 ------- --------
pool  5a9c8adb-44dd-402e-9a67-e5ec9b0d29fd 0       2
    Tier Size   Used   Imbalance
    ---- ----   ----   -------------
    SCM  240 MB 1.9 kB 0%

    Tier Size   Used   Imbalance
    ---- ----   ----   -------------
    NVMe 4GB    10 kB  0%
```